crypto/cmp/cmp_local.h

```diff
@@ -49,6 +49,7 @@ struct ossl_cmp_ctx_st {
     int keep_alive; /* persistent connection: 0=no, 1=prefer, 2=require */
     int msg_timeout; /* max seconds to wait for each CMP message round trip */
     int total_timeout; /* max number of seconds an enrollment may take, incl. */
+    int tls_used; /* whether to use TLS for client-side HTTP connections */
```

Comment:
Presumably, this is not just for clients? In `apps/cmp.c` the flag is set regardless of `opt_server`

Reply:
So far, `tls_used` is implemented for the client side only.
And as now stated in  `openssl-cmp.h.in`:
```
It is ignored if the B<-server> option is not given
```